examples/src/routes/MeasureValueFilter.jsx

```diff
@@ -1,20 +1,68 @@
+import MeasureValueFilterShownInPercentageExampleSRC from "!raw-loader!../components/MeasureValueFilterShownInPercentageExample"; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import MeasureValueFilterStackedToHundredPercentExampleSRC from "!raw-loader!../components/MeasureValueFilterStackedToHundredPercentExample"; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import MeasureValueFilterFormattedInPercentageExampleSRC from "!raw-loader!../components/MeasureValueFilterFormattedInPercentageExample"; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
 
 export const MeasureValueFilter = () => (
     <div>
-        <h1>Filter by Value</h1>
+        <h1>Filter by Measure Value</h1>
         <p>
             Here is how you can filter the entire insight by the value of a measure. The granularity is
             defined by the attributes in the insight.
         </p>
         <hr className="separator" />
-        <ExampleWithSource for={MeasureValueFilterExample} source={MeasureValueFilterExampleSRC} />
+        <h2>Comparison and range filters</h2>
+        <p>Example of filtering of visualization by either one or two measure values.</p>
+        <div className={"s-measure-value-filter-example-1"}>
+            <ExampleWithSource for={MeasureValueFilterExample} source={MeasureValueFilterExampleSRC} />
+        </div>
+        <h2>Filter by measure value shown in %</h2>
+        <p>
+            In the case when visualization is filtered by a measure that is shown in %,{" "}
+            <b>the filter value is in the original measure scale</b> and not a percentage value that is
+            rendered by the visualization.
+        </p>
+        <div className={"s-measure-value-filter-example-2"}>
+            <ExampleWithSource
+                for={MeasureValueFilterShownInPercentageExample}
+                source={MeasureValueFilterShownInPercentageExampleSRC}
+            />
+        </div>
+        <h2>Filter by measure value stacked to 100%</h2>
+        <p>
+            In the case when visualization is filtered by a measure that is stacked to 100%,{" "}
+            <b>the filter value is in the original measure scale</b> and not a percentage value that is
+            rendered by the visualization.
+        </p>
+        <div className={"s-measure-value-filter-example-3"}>
+            <ExampleWithSource
+                for={MeasureValueFilterStackedToHundredPercentExample}
+                source={MeasureValueFilterStackedToHundredPercentExampleSRC}
+            />
+        </div>
+        <h2>Filter by measure value formatted in %</h2>
+        <p>
+            In the case when visualization is filtered by a measure that is formatted in %,{" "}
```

Comment:
...and here: When the visualization is filtered by a measure that is formatted in %, the filter value is in the form of a ratio (for instance 0.5 which is 50%) and not in the percentage scale. <the rest is OK>